bin/templates/project/app/build.gradle

```diff
@@ -30,6 +30,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath 'com.google.gms:google-services:3.2.1'
```

Comment:
Would it be possible to wrap this `classpath` in an if condition, like how the apply is done?
It would be nice to minimize the amount of downloading of build related libraries if they are not needed.

```
if (cdvHelpers.getConfigPreference('EnableGoogleServicesPlugin', 'false').toBoolean()) {
  classpath 'com.google.gms:google-services:3.2.1'
}
```

Reply:
It looks like the object `cdvHelpers` is not available at the dependency scope. Project build failed with the error:
```
Could not get unknown property 'cdvHelpers' for object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```

Reply:
@chemerisuk Yes, it seems `cordova.gradle` would need to be applied within the scope so it becomes available.

```
    dependencies {
        apply from: '../CordovaLib/cordova.gradle'

        classpath 'com.android.tools.build:gradle:3.1.0'

        if(cdvHelpers.getConfigPreference('EnableGoogleServicesPlugin', 'false').toBoolean()) {
            println 'Adding classpath "com.google.gms:google-services:3.2.1"'

            classpath 'com.google.gms:google-services:3.2.1'
        }
    }
```

@dpogue Would you know off-hand if something like this would be problematic? The `cordova.gradle` content seems to be only helper methods. Adding it seems to solve the desire to make the classpaths conditional.

Reply:
I don't know much about Gradle, and my experience consists mostly of swearing at it.

I believe you're correct that `cordova.gradle` is helper methods, but I don't know anything about scoping or when it's safe to import (apply?) stuff.